recipes/starshot_event_content_type/config/node.type.event.yml

```diff
@@ -19,7 +23,7 @@ third_party_settings:
     unpublish_revision: true
 name: Event
 type: event
-description: null
+description: 'Content which describes something happening at a specific time.'
```

Comment:
Let's leave this as `null` for the moment, since this description feels a tad awkward to me. But we can open a follow-up PR to bikeshed it.

Reply:
Change made